app/View/Components/Categories.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\View\Components;
+
+use Illuminate\View\Component;
+
+class Categories extends Component
+{
+    /**
+     * Create a new component instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        //
+    }
+
+    /**
+     * Get the view / contents that represent the component.
+     *
+     * @return \Illuminate\Contracts\View\View|string
+     */
+    public function render()
+    {
+        return view('components.categories');
+    }
```

Comment:
if wala ni gamit na file delete nalang

Reply:
Updated the changes under the Categories Page PR.  

link: https://github.com/framgia/ph_sels_laravel_Angelo/pull/5/commits/eea21bdf39f9253c579177158dfb8cf2e69b6c1d